config.py

```diff
@@ -0,0 +1,11 @@
+# Configurables
```

Comment:
the above pyc files should not be included. find a way to remove them, probably just git rm *.pyc